front/src/modules/events/hooks/useIsTelemetryEnabled.ts

```diff
@@ -0,0 +1,4 @@
+export function useIsTelemetryEnabled() {
+  // TODO: replace by clientConfig
+  return process.env.TELEMETRY_ENABLED !== 'false';
```

Comment:
is there a TELEMETRY_ENALBLED variable in front/.env and server/.env ?
In any case, we should update .env.example

Reply:
It's documented in .doc ; I'm not sure we should overload .example with every parameter ever available, as we want to keep a simple install. E.g. Laravel has many more ENV_XXX you can find in config which are not in their .env.example